### PR TITLE
(SERVER-1549) Set OSS latest job to run automatically

### PR DESCRIPTION
**Pull request by cprice404**
This commit adds a `JobDSL.groovy` file for the "OSS latest" job,
and schedules that job to run automatically, once a week, on
Monday evenings.

### CODE REVIEW

File: jenkins-integration/jenkins-jobs/job_bootstrap.groovy [hunk 1]
```diff
@@ -1,6 +1,54 @@
...
+        // NOTE: HACK.  for some reason, the 'configure' block may get called
+        //  several times.  See https://issues.jenkins-ci.org/browse/JENKINS-39417 .
+        // Worse, it seems like on the subsequent calls to the configure block,
+        // the state of the job is reset to still include the original parameter
+        // data.  It seems harmless to simply execute the overrides multiple times,
+        // but logging the messages about the overrides multiple times looks very
+        // confusing in the seed job output.  By closing over this local variable,
+        // we can make sure the log messages only show up once, which makes
+        // the seed job output a little less confusing.
+        def param_checked = false
```
Comment (camlow325) [inline jenkins-integration/jenkins-jobs/job_bootstrap.groovy]: Will this end up suppressing messages that it shouldn't in the case where we're overriding more than one parameter through the helper?

Comment (camlow325) [inline jenkins-integration/jenkins-jobs/job_bootstrap.groovy]: Ah, okay, got it.  Works for me, then.

**Comment by jpinsonault** [comment]
As far as I understand groovy and jenkins (which isn't far), this seems reasonable

**Comment by jpinsonault** [comment]
👍

**Pull request merged**